internal/description/description.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package description
+
+const (
+	Atlas             = "Atlas operations."
+	Alerts            = "Manage alerts for your project."
+	AcknowledgeAlerts = "Acknowledge an Alert."
+	Config            = "Manage alerts configuration for your project."
+	ConfigLong        = "The configs command provides access to your alerts configurations. You can create, edit, and delete alert configurations."
+	CreateConfig      = "Create an alert configuration for a project."
+	DeleteConfig      = "Delete an alert config."
+	Fields            = "Manage alert configuration fields for your project."
+	FieldsType        = "List alert configurations available field types."
+	ListConfig        = "List alert configurations for a project."
+	UpdateConfig      = "Update an alert configuration for a project."
+	DescribeAlerts    = "Describe an Alert."
+	ListAlerts        = "List Alerts."
+	Backup            = "Manage continuous backups for your project."
+	Checkpoints       = "Manage backup checkpoints for your project."
+	ListCheckpoints   = "List continuous backup checkpoints."
+	Restores          = "Manage restore jobs."
+	ListRestores      = "Start a restore job."
+	StartRestore      = "Start a restore job."
+	Snapshots         = "Manage continuous snapshots for your project."
+	ListSnapshots     = "List continuous snapshots for a project."
+	Clusters          = "Manage Atlas clusters for your project."
+	ClustersLong      = "The clusters command provides access to your cluster configurations. You can create, edit, and delete clusters."
+	CreateCluster     = "Create a MongoDB cluster in Atlas."
+	DeleteCluster     = "Delete an Atlas cluster."
+	DescribeCluster   = "Describe an Atlas cluster."
+	ListClusters      = "List Atlas clusters for a project."
```

Comment:
We could remove atlas from here and use the same const for both atlas and om/cm
if you want to keep both atlas and om separate then lets please rename this to `ListAtlasClusters` so it looks the same as `ListOMClusters`, same for other cluster consts